exec/java-exec/src/test/java/org/apache/drill/BaseTestQuery.java

```diff
@@ -371,6 +373,16 @@ public static void testNoResult(String query, Object... args) throws Exception {
     testNoResult(1, query, args);
   }
 
+  public static void alterSession(String key, Object value) throws Exception {
```

Comment:
There is the setSessionOption method on line 487 that already does something similar. Could we delete those methods and their usages in favor of this one?